libmamba/src/core/run.cpp

```diff
@@ -290,7 +292,7 @@ namespace mamba
         }
 
         LOG_DEBUG << "Currently running processes: " << get_all_running_processes_info();
-        LOG_DEBUG << "Remaining args to run as command: " << join(" ", command);
+        fmt::print(LOG_DEBUG, "Remaining args to run as command: {}", fmt::join(command, " "));
```

Comment:
This is more a general question, but are we using `fmt` now everywhere for logging?
(btw, the `LOG_DEBUG` in the line above could be replaced as well)

Reply:
I think we can use `fmt` wherever we need it, here it avoids allocating the result of `join`.